documentation/source/user_section/tutorials/4d/processing-dmri-data/motion-correction-for-dmri.rst

```diff
@@ -0,0 +1,35 @@
+Motion correction for dMRI images
+#################################
+
+SCT features a complex motion correction algorithm, which is inspired by `[Xu et al., Neuroimage 2013] <https://pubmed.ncbi.nlm.nih.gov/23178538/>`_.
+
+.. TODO: The explanation of the motion correction algorithm and the specific parameters is a little confusing to me. Should we rewrite into a step-by-step process? Make it more brief? Leave some of the explanation for the ``-h`` description?
+
+   The key aspects of this algorithm are as follows:
+
+    * **SliceReg:** Slice-wise registration regularized along the Z direction (based on the function antsSliceRegularizedRegistration from ANTs, and described in [De Leener et al., Neuroimage 2017]).
+    * **Grouping:** there is the possibility to group successive volumes in order to have sufficient SNR to estimate a reliable transformation. If your data are very low SNR you can increase the number of successive images that are averaged into group with the flag ``-g``.
+    * **Iterative average:** after registering a new group to the target image (which is usually the first DWI group), the target is averaged with the newly registered group in order to increase the SNR of the target image.
+    * **Outlier detection:** if a detected transformation is too large, it is ignored and the previous transformation is used instead.
+    * **Masking:** in order to estimate motion of the cord, ignoring the rest of the tissue, there is the possibility to include a mask with the flag ``-m``.
```

Comment:
This info was glossed over in the SCT course for time constraints, so I'm not sure how much should be explained here, and how much should be left for a page dedicated to `sct_dmri_moco` as per https://github.com/spinalcordtoolbox/spinalcordtoolbox/issues/3095.

Reply:
looks great